_posts/2023-07-07-announcing-ballerina-2201.7.0-swan-lake-update-7.md

```diff
@@ -0,0 +1,107 @@
+---
+layout: blog-post
+title: Announcing Ballerina 2201.7.0 (Swan Lake Update 7)
+author: Ballerina Team
+published-date: 7 July 2023
+status: Published
+socialmediaimage: Ballerina-Swan-Lake-GA-Release-banner-02-with-button.png
+permalink: /posts/2023-07-07-announcing-ballerina-2201.7.0-swan-lake-update-7/
+---
+
+<style>.cBlogContent p{white-space: break-spaces !important;}</style>
+
+We are excited to announce the [Ballerina 2201.7.0 (Swan Lake Update 7)](https://ballerina.io/downloads/) release, which adds many new features and improvements to the Ballerina programming language. 
+
+The two main highlights of this release are the support for generating GraalVM native executables and performing aggregation-related operations, which are described below.
```

Comment:
Re: GraalVM support was there even in Update 5 and 6. In my opinion, we are making that support official from Update 7 onwards. I.e., removing the experimental flag. @shafreenAnfar 

The phrase "performing aggregation-related operations" is not precise on its own. We are adding aggregation and grouping capabilities to Ballerina query expressions. Please double-check with the compiler team. @hasithaa @gimantha

Reply:
Updated to `adding aggregation and grouping capabilities to Ballerina query expressions`.